fastparquet/test/test_api.py

```diff
@@ -470,12 +470,8 @@ def test_compression_lz4(tempdir):
         "y": {
             "type": "lz4",
             "args": {
-                "compression_level": 5,
-                "content_checksum": True,
-                "block_size": 0,
```

Comment:
Are these options now obsolete for the (single) block? This is just for curiosity.

Reply:
Yeah, the block compressor doesn't have these options.